firebase-appdistribution/src/main/java/com/google/firebase/appdistribution/FirebaseAppDistribution.java

```diff
@@ -339,6 +350,16 @@ void onActivityDestroyed(@NonNull Activity activity) {
     }
   }
 
+  void onActivityCreated(Activity activity) {
+    if (remakeSignInConfirmationDialog) {
+      remakeSignInConfirmationDialog = false;
+      signInConfirmationDialog.show();
```

Comment:
I'm kind of surprised this works. I figured we'd need to call `showSignInConfirmationDialog` with the new activity instance.

Will this work in all cases? Might be worth trying on multiple API levels.

Reply:
Hm thats true not sure why it was working. It is a different instance of the activity when it gets recreated it is possible that it was due to launching from the application context but changed to match rachel's PR.